src/command.rs

```diff
@@ -23,12 +23,38 @@ pub enum Command {
         #[structopt(long = "scope", short = "s")]
         scope: Option<String>,
     },
+
     #[structopt(name = "pack")]
     /// 🍱  create a tar of your npm package but don't publish! [NOT IMPLEMENTED]
     Pack { path: Option<String> },
+
     #[structopt(name = "publish")]
     /// 🎆  pack up your npm package and publish! [NOT IMPLEMENTED]
     Publish { path: Option<String> },
+
+    #[structopt(name = "login", alias = "adduser", alias = "add-user")]
+    /// 👤  Add a registry user account! (aliases: adduser, add-user) [NOT IMPLEMENTED]
```

Comment:
do we want this to still say not implemented? it looks very implemented here :)

Reply:
oh i see. there are some errant "NOT IMPLEMENTED" on the pack and publish commands... you can just remove that :)

Reply:
Will do - that already confused me :)